src/bits/bit_array.rs

```diff
@@ -117,6 +134,25 @@ macro_rules! bit_array_impl {
                 }
             }
 
+            /// Compares two `BitArray`s by their representational ordering
```

Comment:
I would also suggest to follow up and move this doc to the struct definition. Basically what we're saying here is that bit array uses LSB order to store elements and MSB order to compare two arrays